pages/common/git-daemon.md

```diff
@@ -0,0 +1,16 @@
+# git daemon
+
+> A really simple server for Git repositories.
+> More information: <https://git-scm.com/docs/git-daemon>.
+
+- Git daemon with a whitelisted set of directories:
+
+`git daemon --export-all {{path/to/directory1}} {{path/to/directory2}}`
+
+- Git daemon with a base directory and allow pulling from all directories that look like Git repos:
```

Comment:
Perhaps this will read better?
```suggestion
- Launch a Git daemon with a base directory and allow pulling from all directories that look like Git repos:
```